portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,10 @@
+
+function scrollUp() {
+    window.scrollTo({top:0, behavior: 'smooth'});
+}
+
+async function getGreeting() {
+  const response = await fetch('/data');
+  const greeting = await response.text();
+  document.getElementById('message-container').innerText = greeting;
```

Comment:
We typically avoid directly manipulating the innerText or innerHTML for security reasons.

Suggest instead appending a textNode:

https://www.w3schools.com/jsref/met_node_appendchild.asp